<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_checkEmail</class>
 <widget class="QWidget" name="form_checkEmail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>131</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>401</width>
    <height>131</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>401</width>
    <height>131</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Проверка подключения к почте</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Проверка подключения к электронной почте&lt;/p&gt;&lt;p&gt;Для успешной проверки работоспособности сервиса отправки писем, введите адрес электронной почты, на который нужно отправить тестовое письмо.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLineEdit" name="editEmail">
        <property name="minimumSize">
         <size>
          <width>282</width>
          <height>0</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Адрес электронной почты</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="buttonSend">
        <property name="text">
         <string>Проверить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonSend</sender>
   <signal>clicked()</signal>
   <receiver>form_checkEmail</receiver>
   <slot>buttonSend_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buttonSend_clicked()</slot>
 </slots>
</ui>
